pkg/nsxt/config/config_yaml.go

```diff
@@ -0,0 +1,56 @@
+/*
+ Copyright 2020 The Kubernetes Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+
+	"k8s.io/klog"
+)
+
+// ValidateConfig checks NSXT configurations
+func (cfg *NsxtConfigYAML) ValidateConfig() error {
+	if cfg.VMCAccessToken != "" {
+		if cfg.VMCAuthHost == "" {
+			msg := "vmc auth host must be provided if auth token is provided"
+			klog.Errorf(msg)
```

Comment:
I know this was just copied from the old package but would be good to clean this up a bit. Can we remove these log lines and just return the error
```
return errors.New("vmc auth host...")
```

same with the errors below